### PR TITLE
Expand on rust unsafe usage

### PR DESCRIPTION
**Pull request by g-k**
Changes:
* recommend `#![forbid(unsafe_code)]` if possible
* as @bbangert suggested people should justify unsafe usage

diff ignoring whitespace: https://github.com/mozilla-services/websec-check/pull/14/files

r? @psiinon

### CODE REVIEW

File: rust.md [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
+## Markdown issue template
+For the above checklist.
+
+```
+* [ ] Use https://github.com/RustSec/cargo-audit to check for dependency security vulnerabilities
+* [ ] Use https://github.com/rust-lang/rust-clippy for linting (easier for new projects)
+* [ ] Use https://github.com/mozilla-services/slog-mozlog-json for service logging
+* [ ] Binaries (as opposed to libraries) should include a Cargo.lock file in version control
+* [ ] Use https://dependabot.com/rust/ to keep your dependencies up to date
+* [ ] Add `#![forbid(unsafe_code)]` to your crate (e.g. the top of `main.rs` or `lib.rs`)
```
Comment (g-k) [inline rust.md:42]: this makes https://crates.io/crates/cargo-geiger happy too

**Comment by psiinon** [comment]
r+ but might conflict with #12 :)

**Comment by g-k** [comment]
Yeah land #12 first and I'll rebase

**Comment by psiinon** [comment]
I've merged #12 :)

**Pull request merged**